templater/jobs/presubmit/eks-anywhere-build-tooling/eks-a-upgrader-image-presubmits.yaml

```diff
@@ -0,0 +1,13 @@
+jobName: eks-a-upgrader-image-presubmits
+runIfChanged: EKS_DISTRO_MINIMAL_BASE_TAG_FILE|EKSD_LATEST_RELEASES|^build/lib/.*|projects/aws/upgrader/.*|projects/containerd/.*|projects/kubernetes-sigs/cri-tools/.*
```

Comment:
the tag file should the glibc one and no need to watch the containerd/critools pojects